### PR TITLE
feat: Add `strikethrough` and update `underline` icons

### PR DESCRIPTION
**Pull request by ericfennis**
Closes [#161](https://github.com/feathericons/feather/issues/161)
And contributes to: [#766](https://github.com/feathericons/feather/issues/766)

image

100% 
image

Updated the underline icon to match same height from other text-editor icons.
image

### CODE REVIEW

File: icons/strikethrough.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M16 4H9A3 3 0 0 0 6.17 8" />
+    <path d="M14 12a4 4 0 0 1 0 8H6" />
+    <line x1="4" y1="12" x2="20" y2="12" />
```
Comment (jletey) [inline icons/strikethrough.svg:14]: Tab size of 2 😄

```suggestion
  <path d="M16 4H9A3 3 0 0 0 6.17 8" />
  <path d="M14 12a4 4 0 0 1 0 8H6" />
  <line x1="4" y1="12" x2="20" y2="12" />
```

File: icons/underline.svg [hunk 2]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M6 3v7a6 6 0 0 0 6 6 6 6 0 0 0 6-6V3" />
-  <line x1="4" y1="21" x2="20" y2="21" />
+    <path d="M6,4v6a6,6,0,0,0,12,0V4" />
+    <line x1="4" y1="20" x2="20" y2="20" />
```
Comment (jletey) [inline icons/underline.svg:13]:
```suggestion
  <path d="M6,4v6a6,6,0,0,0,12,0V4" />
  <line x1="4" y1="20" x2="20" y2="20" />
```

**Comment by locness3** [comment]
Looks nice

**Comment by ericfennis** [comment]
@jletey Thanks for your sharpness, I added the spaces!

**Pull request merged**